### PR TITLE
Clean

### PR DESCRIPTION
**Pull request by jairhenrique**

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -1,7 +1,7 @@
 Ramos
 =====
 
-.. image:: https://travis-ci.org/luizalabs/ramos.svg?branch=master
+.. image:: https://travis-ci.org/luizalabs/ramos.svg
```
Comment (drgarcia1986) [inline README.rst:4]: if a pr fails on CI what status will show here?

File: tests/test_pool.py [hunk 2]
```diff
@@ -28,59 +25,52 @@ def __init__(self, *args, **kwargs):
         self.kwargs = kwargs
 
 
-class TestBackendPool(object):
+def configure_pool():
```
Comment (drgarcia1986) [inline tests/test_pool.py:28]: what about a fixture ?

File: README.rst (hunk 1 above)
Comment (jairhenrique) [reply README.rst:4]: I will revert this.

File: tests/test_pool.py (hunk 2 above)
Comment (jairhenrique) [reply tests/test_pool.py]: @drgarcia1986 done.

File: tests/test_pool.py [hunk 3]
```diff
@@ -28,60 +25,49 @@ def __init__(self, *args, **kwargs):
...
+        assert issubclass(backends[0], FakeXYZBackend)
+        assert issubclass(backends[1], FakeABCBackend)
+
+    def test_get_backend_should_return_a_backend_class(self, configured_pools):
         backend = BackendPool.get_class('fake_abc')
         assert issubclass(backend, FakeABCBackend)
 
     def test_iterator_should_return_an_interator_with_all_backends_instances(
-        self
+        self, configured_pools
```
Comment (rafaelgotts) [inline tests/test_pool.py:69]:
```suggestion
        self,
        configured_pools
```

File: tests/test_pool.py [hunk 4]
```diff
@@ -92,21 +78,8 @@ def test_iterator_should_return_an_interator_with_all_backends_instances(
         assert isinstance(backend_1, FakeABCBackend)
 
     def test_classes_iterator_should_return_an_interator_with_all_backends_classes(  # noqa
-        self
+        self, configured_pools
```
Comment (rafaelgotts) [inline tests/test_pool.py:81]:
```suggestion
        self,
        configured_pools
```

File: tests/test_pool.py (hunk 3 above)
Comment (drgarcia1986) [reply tests/test_pool.py:69]: I don't follow, maybe we can use [black](https://github.com/psf/black) for ramos and according to black the @jairhenrique ways is correct

Comment (rafaelgotts) [reply tests/test_pool.py:69]: I don't know black framework yet. I just follow another arguments pattern on project

**Comment by jairhenrique** [comment]
@drgarcia1986 @rafaelgotts #25

File: .gitignore [hunk 5]
```diff
@@ -86,4 +86,5 @@ Icon
 .AppleDesktop
 Network Trash Folder
 Temporary Items
-.apdisk
\ No newline at end of file
+.apdisk
+.vscode/
```
Comment (renanivo) [inline .gitignore:90]: What about using your [global gitignore](https://help.github.com/pt/github/using-git/ignoring-files#configuring-ignored-files-for-all-repositories-on-your-computer) for editor files? As there are several editors and new ones are created, this behavior avoid a ripple effect of changes when people adopt different editors.

Comment (jairhenrique) [reply .gitignore]: :+1:

**Comment by jairhenrique** [comment]
@drgarcia1986 @renanivo I no longer have permission to merge pull requests. Fell free to do that 😄

**Pull request merged**